get.md

```diff
@@ -0,0 +1 @@
+Get.md file
```

Comment:
[GET] (get.md) to retrieve data from the server, POC in this case the bill, once the order has been placed.